src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -181,6 +188,7 @@ def kubernetes_exception_handler(ex, fault_type, summary, error_message='Error o
             telemetry.set_exception(exception=ex, fault_type=fault_type, summary=summary)
             raise CLIError(error_message + "\nError Response: " + str(ex.body))
     else:
+        telemetry.set_user_fault()
```

Comment:
>set_user_fault [](start = 18, length = 14)

shouldnt we just set the user fault here blindly? for 200 status code, it shouldnt come in handler

Reply:
Yes, that makes sense. Added